specify/specify7
# Add COG children picker to forms

Fixes #5185

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- go to data entry and select Collection Object Group 
- define a name and select a cogType 
- [ ] verify you can create a new CO child and save the CO 
- [ ] verify you can save the main COG form 
- [ ] verify after refresh the CO child is still there 
- [ ] verify you can create a new COG child and save the COG 
- [ ] verify you can save the main COG form 
- [ ] verify after refresh the COG child is still there 
- [ ] verify you can add one CO child using query builder
- [ ] verify you can save the main COG form 
- [ ] verify after refresh the CO child is still there 
- [ ] verify you can add several CO children using query builder
- [ ] verify you can save the main COG form 
- [ ] verify after refresh the CO child is still there
- [ ] verify you can add one COG child using query builder
- [ ] verify you can save the main COG form 
- [ ] verify after refresh the COG child is still there 
- [ ] verify you can add several COG children using query builder
- [ ] verify you can save the main COG form 
- [ ] verify after refresh the COG child is still there
- [ ] verify you can delete any children 
- [ ] verify you can uncollapse each child 
- you should not be able to add a COGChild to a COG that is already in use in another COG, when doing so an error should be thrown saying that the COGChild is already used 

This PR affects database migrations. See migration testing instructions ([here](https://specify.github.io/testing/pull_requests.html))


View Def  : [geo.common.views.xml.zip](https://github.com/user-attachments/files/17590977/better.common.views.xml.zip)
